tests/generic_views/views.py

```diff
@@ -66,6 +66,15 @@ class AuthorList(generic.ListView):
     queryset = Author.objects.all()
 
 
+class AuthorListGetQueryset(AuthorList):
+    def get_queryset(self):
+        """
+        Simulate a get_queryset with conditions that don't match anything,
```

Comment:
How about `An incorrect implementation of get_queryset.`

Reply:
You don't really need a docstring at all, IMO. The test itself explains it well enough.